ambry-account/src/main/java/com/github/ambry/account/HelixAccountServiceFactory.java

```diff
@@ -62,18 +60,15 @@ public AccountService getAccountService() {
     logger.info("Starting a HelixAccountService");
     ZkClient zkClient = new ZkClient(storeConfig.zkClientConnectString, storeConfig.zkClientSessionTimeoutMs,
         storeConfig.zkClientConnectionTimeoutMs, new ZNRecordSerializer());
-    List<String> subscribedPaths = Collections.singletonList(storeConfig.rootPath);
     HelixPropertyStore<ZNRecord> helixStore =
-        new ZkHelixPropertyStore<>(new ZkBaseDataAccessor<>(zkClient), storeConfig.rootPath, subscribedPaths);
```

Comment:
where is the subscription instead? or is it not required anymore?

Reply:
`HelixPropertyStore` has its own cache, and it listens to the change of ZK. For any reason, if FE receives the notification for account change is received earlier than `HelixPropertyStore` receives the notification for data update in ZK, or earlier than the update actually propagated to the `HelixPropertyStore`, FE will read the outdated accounts though they are already updated in ZK.

There is no explicit way to disable the cache for `HelixPropertyStore`. Per the Helix team suggestion, removing the subscribed paths meaning that do not listen to the changes for these paths, which implicitly makes reading these paths directly fetch from remote.